docs/reference/requirement-sets/excel-api-requirement-sets.md

```diff
@@ -52,11 +52,200 @@ For more information about versions, build numbers, and Office Online Server, se
+| Workbook [Save](../../excel/excel-add-ins-workbooks.md#save-the-workbook-preview) and [Close](../../excel/excel-add-ins-workbooks.md#close-the-workbook-preview) | Save and close workbooks.  | [Workbook](/javascript/api/excel/excel.workbook) |
+| [Insert Workbook](../../excel/excel-add-ins-workbooks.md#insert-a-copy-of-an-existing-workbook-into-the-current-one-preview) | Insert one workbook into another.  | [Workbook](/javascript/api/excel/excel.worksheetcollection) |
+
+The following is a complete list of APIsc currently in preview.
+
+| Class | Fields | Description |
+|:---|:---|:---|
+|[Comment](/javascript/api/excel/excel.comment)|[content](/javascript/api/excel/excel.comment#content)|Get or set the content.|
+||[delete()](/javascript/api/excel/excel.comment#delete--)|Deletes the comment thread.|
+||[getLocation()](/javascript/api/excel/excel.comment#getlocation--)|Get location of the comment.|
+||[authorEmail](/javascript/api/excel/excel.comment#authoremail)|Get author email of the comment.|
+||[authorName](/javascript/api/excel/excel.comment#authorname)|Get author name of the comment.|
+||[creationDate](/javascript/api/excel/excel.comment#creationdate)|Get creation time of the comment. Will return null if the comment is converted from note, as in this case, the comment will not has created date.|
+||[id](/javascript/api/excel/excel.comment#id)|Represents the comment identifier. Read-only.|
+||[replies](/javascript/api/excel/excel.comment#replies)|Represents a collection of reply objects associated with the comment. Read-only.|
+|[CommentCollection](/javascript/api/excel/excel.commentcollection)|[add(content: string, cellAddress: Range \| string, contentType?: "Plain")](/javascript/api/excel/excel.commentcollection#add-content--celladdress--contenttype-)|Creates a new comment(comment thread) based on the cell location and content. Invalid argument will be thrown if the location is larger than one cell.|
+||[add(content: string, cellAddress: Range \| string, contentType?: Excel.ContentType)](/javascript/api/excel/excel.commentcollection#add-content--celladdress--contenttype-)|Creates a new comment(comment thread) based on the cell location and content. Invalid argument will be thrown if the location is larger than one cell.|
+||[getCount()](/javascript/api/excel/excel.commentcollection#getcount--)|Gets the number of comments in the collection.|
+||[getItem(commentId: string)](/javascript/api/excel/excel.commentcollection#getitem-commentid-)|Returns a comment identified by its ID. Read-only.|
+||[getItemAt(index: number)](/javascript/api/excel/excel.commentcollection#getitemat-index-)|Gets a comment based on its position in the collection.|
+||[getItemByCell(cellAddress: Range \| string)](/javascript/api/excel/excel.commentcollection#getitembycell-celladdress-)|Gets a comment on the specific cell in the collection.|
```

Comment:
For the specified cell, get all comments? I'm confused by the name - it uses "item", but it belongs to the collection object?